src/Symfony/Bundle/SecurityBundle/EventListener/AclSchemaListener.php

```diff
@@ -11,13 +11,17 @@
 
 namespace Symfony\Bundle\SecurityBundle\EventListener;
 
+@trigger_error(sprintf('Class "%s" is deprecated since version 3.4 and will be removed in 4.0. Use Symfony\Bundle\AclBundle\EventListener\SetAclCommand instead.', AclSchemaListener::class), E_USER_DEPRECATED);
```

Comment:
looks like a copy paste error on the alternative class

Reply:
Fixed in https://github.com/symfony/symfony/pull/24352